test/transcoding/IntelFPGAMemoryAttributes.ll

```diff
@@ -168,6 +168,9 @@
 ;   } s;
 ;   s.field = 0;
 ; }
+; void static_memory_attribute() {
+;   [[intelfpga::register]] int register_var[32];
+; }
```

Comment:
A bit confused with the naming - we have an `IntelFPGAMemoryAttributesForStaticVar.ll` test for static memory attributes.